src/EnergyPlus/UnitarySystem.cc

```diff
@@ -523,6 +523,55 @@ namespace UnitarySystems {
                 if (this->m_CoolingCoilType_Num == DataHVACGlobals::CoilDX_CoolingTwoSpeed) {
                     DXCoils::SetCoilSystemCoolingData(state, this->m_CoolingCoilName, this->Name);
                 }
+                // open this up to include UnitarySystem on a future commit
+                if (!this->m_FanExists && this->UnitType == "CoilSystem:Cooling:DX") {
+                    if (this->m_CoolCoilExists) { // copy this block for heating coil also on future commit
+                        bool coilDataSet = false;
+                        bool errorsFound = false;
+                        std::string coilName;
+                        std::string coilType;
+                        if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed) {
+                            VariableSpeedCoils::getCoilTypeAndName(state, this->m_CoolingCoilIndex, coilType, coilName, errorsFound);
+                            if (!errorsFound) coilDataSet = true;
+                        } else if (this->m_CoolingCoilIndex == DataHVACGlobals::CoilDX_CoolingSingleSpeed) {
+                            // need to check for all other coil types
```

Comment:
HVACDXSystem only did this for VS coils. Will open this up in a future pass.

        if (!state.dataHVACDXSys->DXCoolingSystem(DXSystemNum).VSCoilFanInfoSet && AirLoopNum > 0) {
            if (state.dataHVACDXSys->DXCoolingSystem(DXSystemNum).CoolingCoilType_Num == Coil_CoolingAirToAirVariableSpeed) {

Reply:
DesiccantDehumidifierWithAirToAirCoil tabular output now reports the fan associated with the DXSystem.

![image](https://user-images.githubusercontent.com/5778156/126000799-cb13e151-d113-4361-b0f4-6205a3ef70ad.png)

There are some files not reporting this information and now I know why. E.g., UnitarySystem_DXCoilSystemAuto which uses a UnitarySystem to mimic the CoilSystem file DXCoilSystemAuto.

![image](https://user-images.githubusercontent.com/5778156/126001364-2a4baba0-5fe0-4cd6-9664-86e6967e35e7.png)

Reply:
Great!!